utils/mil_tools/pythonProgramFinder/pythonFinder.py

```diff
@@ -0,0 +1,59 @@
+import os
```

Comment:
This script should be executable using ```rosrun mil_tools pythonFinder.py```. For this it should:
1) be given execute permissions ```chmod +x pythonFinder.py```
1) be moved to ```mil_tools/scripts```
1) Start with the magic first line ```#!/usr/bin/env python```

Note that we currently don't use python 3 in our projects. The program should be able to run with python 2 (almost the same)